JSON_for_IO-Link.yaml

```diff
@@ -56,15 +56,154 @@ tags:
   - name: masters
     description: Access to parameters of the IO-Link master itself
   - name: tracks
-    description: Access to the tracks (radios) of the IO-Link wireless master    
+    description: Access to the tracks (radios) of the IO-Link wireless master
   - name: ports
     description: Access to the IO-Link ports of the IO-Link master
   - name: devices
     description: Access to the IO-Link Devices connected ot the Master
 
-    
 paths:
-################################################################################
+  ################################################################################
+  '/openapi':
+    get:
+      tags:
+        - general
+      operationId: getOpenApiDocument
+      summary: >-
+        Get the OpenAPI interface description of the IO-Link Master.
+      description: >-
+        Get the REST interface description of the IO-Link API as OpenAPI YAML
+        document.
+      responses:
+        '200':
+          description: Successful operation
+          content:
+            application/text:
+              schema:
+                type: string
+              examples:
```

Comment:
I propose to add a third example 'url-local', thus pointing to a yaml file inside the device.